internal/proto/milvus.proto

```diff
@@ -268,32 +268,62 @@ message CreatePartitionRequest {
   string partition_name = 4;
 }
 
+/*
+* Drop partition in created collection.
+*/
 message DropPartitionRequest {
-  common.MsgBase base = 1; // must
+  // Not useful for now
+  common.MsgBase base = 1;
+  // Not useful for now
   string db_name = 2;
-  string collection_name = 3; // must
-  string partition_name = 4; // must
+  // The collection name in milvus
+  string collection_name = 3;
+  // The partition name you want to drop
+  string partition_name = 4; 
 }
 
+/*
+* Check partition exist in collection or not.
```

Comment:
Check if partition exist in collection or not.

Reply:
Sorry for fix this slowly, thanks for advice